crates/typst-library/src/math/matrix.rs

```diff
@@ -110,6 +125,26 @@ pub struct MatElem {
     #[fold]
     pub augment: Option<Augment>,
 
+    /// The gap between rows.
```

Comment:
Could you add a third setting `gap` that is a shorthand for path at once? You can take a look at `grid` to see how this works.